### PR TITLE
chore(actions): Don't hardcode packages to release in release action

### PR DESCRIPTION
**Pull request by markflorkowski**

### CODE REVIEW

File: .github/workflows/release-canary.yaml [hunk 1]
```diff
@@ -50,10 +50,11 @@ jobs:
       - name: Authenticate to npm and publish
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > .npmrc
-          pnpm publish packages/uploadthing --access public --tag canary --no-git-checks
-          pnpm publish packages/mime-types --access public --tag canary --no-git-checks
-          pnpm publish packages/shared --access public --tag canary --no-git-checks
-          pnpm publish packages/react --access public --tag canary --no-git-checks
+          for dir in packages/*/; do
+            if [ "$dir" != "packages/config/" ]; then
+              pnpm publish "$dir" --access public --tag canary --no-git-checks
```
Comment (juliusmarminge) [inline .github/workflows/release-canary.yaml:55]: Will $dir include packages/ here?

Comment (markflorkowski) [reply .github/workflows/release-canary.yaml:55]: If you look at the previous runs, it seems to be working, so yes? 🤷‍♂️

Comment (juliusmarminge) [reply .github/workflows/release-canary.yaml:55]: yup!

**Review by juliusmarminge — APPROVED** [review]
Made it so the comment prints out all packages @markflorkowski

**Pull request merged**